bootstraptest/test_yjit.rb

```diff
@@ -5044,6 +5050,9 @@ def compiled_method(is_private)
     begin
       obj.compiled_method(true)
     rescue
+      # expected
```

Comment:
Something like this would highlight the `raise warn` typo above
```suggestion
      raise unless $!.message =~ /shape variations/
```